### DISCUSSION TITLE
undefined external stimuli on cognition

### DISCUSSION THREAD

**Discussion by mayablume** [jspsych/jsPsych#2347] [Q&A] [answered]
Hi everyone! 

I'm trying to host a jsPysch experiment on cognition.run. It was working well on my computer, but no luck once hosted. It seems like it's not recognizing my uploaded stimuli. I've been following [this thread ](https://github.com/jspsych/jsPsych/discussions/1227) but it hasn't solved my problem. 

I'm attaching my code and the stimuli file. Thanks for your help!

[stimuli.txt](https://github.com/jspsych/jsPsych/files/7585613/stimuli.txt)
 
[experiment_code.txt](https://github.com/jspsych/jsPsych/files/7585615/experiment_code.txt)

**Comment by jodeleeuw** [comment, marked as answer]
Hi @mayablume,

Are you loading the `stimuli.js` file in the External JS/CSS part of the editor?

![image](https://user-images.githubusercontent.com/595524/143078232-30c870f8-5908-4c04-87f2-ae1f53940f11.png)

Once you do that you shouldn't need the `var test_stimuli = "stimuli.js"` line at the top of your code.

**Reply by mayablume** [reply]
Thank you! It works now.
